packages/engine/src/gltf/GLTFState.tsx

```diff
@@ -153,6 +158,59 @@ export const GLTFSourceState = defineState({
   }
 })
 
+export const GLTFLoadSystem = defineSystem({
+  uuid: 'ee.engine.gltf.GLTFLoadSystem',
+  insert: { after: PresentationSystemGroup },
+  reactor: () => {
+    const gltfSimulationEntities = useQuery([GLTFComponent])
+    const gltfAuthoringEntities = useQuery([GLTFComponent], Layers.Authoring)
+    return (
+      <>
+        {/* The authoring layer entities will have their entities propagated to the simulation layer */}
+        {gltfSimulationEntities.map((entity) => (
+          <GLTFComponentReactor key={'simulation-' + entity} entity={entity} />
+        ))}
+        {gltfAuthoringEntities.map((entity) => (
+          <GLTFAuthoringComponentReactor key={'authoring-simulation-' + entity} entity={entity} />
+        ))}
+        {gltfAuthoringEntities.map((entity) => (
+          <GLTFComponentReactor key={'authoring-' + entity} entity={entity} />
```

Comment:
authoring layer and simulation layer gltf component entities should not both have reactive logic running. This will cause the same data to be loaded twice instead of loaded once and then propagated according to layer relations. In the original implementation this is done in the gltf component reactor where it checks if an upstream entity exists and defers load responsibility upstream if so